### PR TITLE
add mqtt support

### PR DESCRIPTION
**Pull request by py4mac**

### CODE REVIEW

**Comment by k33g** [comment]
👋 @py4mac 
I get strange behaviour. I did the tests with 2 MQTT brokers: Mosquitto and [Mochi](https://github.com/mochi-co/mqtt), locally on an MBP M1 and with GitPod.

### Test 1 (locally + GitPod + 2 MQTT brokers)

- Launch the MQTT broker
- Build capsule-mqtt-subscriber & run capsule-mqtt-subscriber (listening `topic/sensor0`)
- Publish message with a Node.js MQTT publisher on the `topic/sensor0`
- Everything is ok; capsule-mqtt-subscriber is triggered 🟢
- Build capsule-mqtt-publisher & run capsule-mqtt-publisher (on the `topic/sensor0`)
- capsule-mqtt-subscriber is not triggered 🔴
- If I rerun the Node.js MQTT publisher, capsule-mqtt-subscriber is not triggered 🔴
- I stop it and restart it, rerun the Node.js MQTT publisher, capsule-mqtt-subscriber is triggered 🟢

### Test 2 (locally + GitPod + 2 MQTT brokers)

- Launch the MQTT broker
- Run a Node.js MQTT subscriber listening on `topic/sensor0`
- Publish message with a Node.js MQTT publisher on the `topic/sensor0`
- Everything is ok; the Node.js MQTT subscriber is triggered 🟢
- Run capsule-mqtt-publisher (on the `topic/sensor0`)
- Everything is ok; the Node.js MQTT subscriber is triggered 🟢
- Do it again several times; all is ok 🟢

[mqtt-js-client.zip](https://github.com/bots-garden/capsule/files/9637921/mqtt-js-client.zip)
[mochi.zip](https://github.com/bots-garden/capsule/files/9637922/mochi.zip)

**Comment by py4mac** [comment]
Thanks for your feedback ;)
Yes you're right we got a constraint that subscriber and publisher must not share the same client_id, it must be unique.
I fixed it inside mqtt wasm examples.

File: wasm_modules/capsule-mqtt-subscriber/go.mod [hunk 1]
```diff
@@ -0,0 +1,11 @@
+module github.com/bots-garden/capsule/wasm_modules/capsule-nats-subscriber
```
Comment (k33g) [inline wasm_modules/capsule-mqtt-subscriber/go.mod:1]: Change with `capsule-mqtt-subscriber`

**Review by k33g — APPROVED** [review]

File: wasm_modules/capsule-mqtt-subscriber/go.mod (hunk 1 above)
Comment (k33g) [reply wasm_modules/capsule-mqtt-subscriber/go.mod:1]: I will change it when preparing the next release

**Comment by k33g** [comment]
👏 thank you @py4mac 😃 -> merged -> v0.2.5 🦖 [T-Rex] 🚢

**Pull request merged**